mandatory/1-fix-functions.js

```diff
@@ -10,31 +10,26 @@
 
 */
 
-function mood() {
+function mood(a) {
```

Comment:
You could just delete the declaration
```suggestion
function mood(isHappy) {
  // let isHappy = true;

  if (isHappy) {
    return "I am happy";
  } else {
    return "I am not happy";
  }
}
```